pkg/services/serviceaccounts/api/api.go

```diff
@@ -89,6 +90,18 @@ func (api *ServiceAccountsAPI) UpgradeServiceAccounts(ctx *models.ReqContext) re
 	}
 }
 
+func (api *ServiceAccountsAPI) ConvertToServiceAccount(ctx *models.ReqContext) response.Response {
+	keyId, err := strconv.ParseInt(web.Params(ctx.Req)[":keyId"], 10, 64)
+	if err != nil {
+		return response.Error(http.StatusBadRequest, "keyId is invalid", err)
+	}
+	if err := api.store.ConvertToServiceAccounts(ctx.Req.Context(), []int64{keyId}); err == nil {
+		return response.Success("service accounts converted")
+	} else {
+		return response.Error(500, "Internal server error", err)
+	}
+}
```

Comment:
we only expose to convert one apikey for now, but backend supports multiple incase of us wanting to support to convert multiple, than we can add that endpoint to the api here.